src/test/mutation/highlights-delete.integration.ts

```diff
@@ -0,0 +1,69 @@
+import { ApolloServer } from 'apollo-server-express';
+import { getServer } from '../../server';
+import sinon from 'sinon';
+import { ContextManager } from '../../context';
+import { readClient } from '../../database/client';
+import { seedData } from '../query/highlights-fixtures';
+import { DELETE_HIGHLIGHT } from './highlights-mutations';
+import { HighlightEntity } from '../../types';
+
+describe('Highlights creation', () => {
```

Comment:
```suggestion
describe('Highlights deletion', () => {
```

my copy-paste probably :)